tests/testsuite_default_SharedObject.py

```diff
@@ -0,0 +1,157 @@
+        self.sharedobject_test_template("array_partial_doubleinit", 1, "--param=object_type:array --param=num_entities:12 --param=full_initialization:false --param=double_initialize:true")
+
+    def test_SharedObject_array_late_initialize(self):
+        self.sharedobject_test_template("array_late_initialize", 1, "--param=object_type:array --param=num_entities:12 --param=late_initialize:true")
+
+
+    # SharedMap Tests
+    # Full Initialization
+    #   single - only ID 0 initializes array
+    #   multi - also has ID N-1 initialize array
+    #     no conflict - both IDs write same data
+    #     conflict - IDs write different data
+    # Partial initialize (everyone initializes their own portion)
+    #   late_write = true
+    #   do not publish = true
+    #   nopub and late_write true
+    #   double_initialize
+    def test_SharedObject_map_full_single(self):
+        self.sharedobject_test_template("map_full_single", 0, "--param=object_type:map --param=num_entities:12 --param=full_initialization:true")
+
+    def test_SharedObject_map_full_multi(self):
+        self.sharedobject_test_template("map_full_multi", 0, "--param=object_type:map --param=num_entities:12 --param=full_initialization:true --param=multiple_initializers:true")
+
+    def test_SharedObject_map_full_multi_conflict(self):
+        self.sharedobject_test_template("map_full_multi_conflict", 1, "--param=object_type:map --param=num_entities:12 --param=full_initialization:true --param=multiple_initializers:true --param=conflicting_write:true")
+
+    def test_SharedObject_map_partial(self):
+        self.sharedobject_test_template("map_partial", 0, "--param=object_type:map --param=num_entities:12 --param=full_initialization:false")
+
+    def test_SharedObject_map_partial_nopub(self):
+        self.sharedobject_test_template("map_partial_nopub", 0, "--param=object_type:map --param=num_entities:12 --param=full_initialization:false --param=publish:false")
+
+    def test_SharedObject_map_partial_late(self):
+        self.sharedobject_test_template("map_partial_late", 1, "--param=object_type:map --param=num_entities:12 --param=full_initialization:false --param=late_write:true")
+
+    def test_SharedObject_map_partial_late_nopub(self):
+        self.sharedobject_test_template("map_partial_late_nopub", 1, "--param=object_type:map --param=num_entities:12 --param=full_initialization:false --param=late_write:true --param=publish:false")
+        
+    def test_SharedObject_map_partial_doubleinit(self):
+        self.sharedobject_test_template("map_partial_doubleinit", 1, "--param=object_type:map --param=num_entities:12 --param=full_initialization:false --param=double_initialize:true")
+        
+#####
+
+    def sharedobject_test_template(self, testtype, exp_rc, options):
+        testsuitedir = self.get_testsuite_dir()
+        outdir = test_output_get_run_dir()
+
+        model_options = '--model-options="{0}"'.format(options)
+        print(model_options)
```

Comment:
change

```print(model_options)``` to  ```log_debug(model_options)```  to the the run clean and only display the model_options during a debug run.

Reply:
Print not needed at all, just leftover from debugging.  Removed.